docs/contributing/3.0-development/plugin-layout.rst

```diff
@@ -84,3 +84,12 @@ of a plugin app, and so on.
 
 This matching of module names is required for the Pulp Platform to be able to auto-discover
 plugin components, particularly for both models and viewsets.
+
+
+Packaging
+---------
+
+The Plugin API is available from PyPI as pulpcore-plugin. A plugin writer needs to specify the
+version of pulpcore-plugin their plugin is dependent on. A plugin writer does not need to be
+concerned which version of pulpcore would work with their plugin since pulpcore-plugin will
+resolve the pulpcore dependency.
```

Comment:
Since this documentation is in the Pulp3 docs, I don't think it's necessary to declare it here. Maybe a line in the release notes? Something like: The PluginAPI currently works with this version of Platform?